tests/dummy/config/optional-features.json

```diff
@@ -0,0 +1,3 @@
+{
+  "jquery-integration": true
```

Comment:
I removed jQuery from the addon a while back, so I think this should prob be `false`

Reply:
ok, cool!